DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/EndToEndTest.java

```diff
@@ -42,6 +42,7 @@ private static Stream<Arguments> scenarios() {
                 scenarios.add(Arguments.of(sourceCluster, targetCluster));
             }
         }
+        scenarios.add(Arguments.of(SearchClusterContainer.OS_LATEST, SearchClusterContainer.OS_LATEST));
```

Comment:
Lets hardcode these values to the `OS_V2_19_1` version, unless that alias is really refers to the latest 2.x version, in which case lets adjust the naming.

Reply:
Good point, it will likely eventually be a 3.x verison